### ISSUE TITLE
PRJNA596980-sra raw data

### ISSUE THREAD

**Issue by kkjtmac** [atomadam2/Liver_Deconvolution#1]
Hi,
I tried to download the PRJNA596980 raw data from SRA, but I faced one problem about the data when running the cellranger count：
First, I downloaded the SRR10744530 sra file, and then fastq-dump --gzip --split-files SRR10744530.1, I got two fastq.gz files, and I checked the file: one file median length is 8 (Index?), another is 98(Read2), so I doubted that Read 1 file is absent as I run the cellranger count, the warning: The read lengths are incompatible with all the chemistries for Sample SRR10744530.
So could you please check the data in SRA? 
Thanks!

**Comment by atomadam2** [comment]
Hey, thanks for the comment. I haven't actually tried to download my own data after uploading to SRA. But I have had issues downloading other people's data from SRA. But I normally get a different problem, in that I usually only get 1 file, that fastq-dump is unable to split.

When I click on data access in the following link, I see all the correct files that should be there. So how are you downloading the data?

https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR10744530

**Comment by kkjtmac** [comment]
Hallo!

From the link you shared, I chose to download the files from "Data access"-AWS-https://sra-pub-run-odp.s3.amazonaws.com/sra/SRR10744530/SRR10744530,
I got the SRR10744530 file and "fastq-dump --gzip --split-files SRR10744530", 
then I got two files and checked the files using "zless -SN SRR10744530_1.fastq.gz |head", 
one file length is 8 (Index 1 from 10x information), another is 98 (Read2), so Read1 file is missing. According the paper using 10x Chromium v3.1, the complete files should be like this:
    - Sample1_S1_L001_I1_001.fastq.gz （length =8）
    - Sample1_S1_L001_R1_001.fastq.gz  （length =28）
    - Sample1_S1_L001_R2_001.fastq.gz （length =98）
 And I also tried to run the cellranger count by changing the file name into SRR10744530_S1_L001_R1_001.fastq.gz, SRR10744530_S1_L001_R2_001.fastq.gz, but it didn't work. 
So what's your suggestion about it? From my side, the Read1 file is missing, so I can not continue the analysis.
Thanks!

**Comment by atomadam2** [comment]
Just to be sure, are your file sizes correct? In other words, is everything downloading correctly?

From my end the data appears to have been uploaded correctly, so I will try to download it and see if it matches.

Thanks for letting me know! And keeping me on my toes.

**Comment by kkjtmac** [comment]
Yes, I tried all the links in SRA or sra-explore. The same problem occurred.
Thanks for your reply!

**Comment by kkjtmac** [comment]
Hallo, 
Did you checked the data in SRA？ I tried several times but it didn't work. And it is possible that you can share the processed rds files？
Thanks！

**Comment by atomadam2** [comment]
Sorry, been out for a few days. I can't figure out what's wrong with SRA. I asked a colleague to see if he can independently download the data and make sure the read lengths are correct.

I'm going to upload the Seurat-ready files to GEO. I don't think I can share the .rds file, as it's really big.

**Comment by kkjtmac** [comment]
> Sorry, been out for a few days. I can't figure out what's wrong with SRA. I asked a colleague to see if he can independently download the data and make sure the read lengths are correct.
> 
> I'm going to upload the Seurat-ready files to GEO. I don't think I can share the .rds file, as it's really big.

Hi, It would be great if the data are kept in GEO!  Thanks!

**Comment by atomadam2** [comment]
I have now double checked a few things and everything seems to be correct in SRA. I will say, downloading data from SRA is and always will be a challenge. Especially single cell data. Not all systems are set up to even be able to do it. It's really annoying. Just out of curiosity, are you able to download other datasets correctly? In the past I have had issues where downloading single-cell data didn't work, but bulk RNA or DNA data worked fine.

I will upload to GEO, but I have never done it before so I need to figure out. I just filled out metadata and am figuring out how to get the data uploaded in an organized way. In the future, I will try to do both, GEO and SRA.

**Comment by kkjtmac** [comment]
> I have now double checked a few things and everything seems to be correct in SRA. I will say, downloading data from SRA is and always will be a challenge. Especially single cell data. Not all systems are set up to even be able to do it. It's really annoying. Just out of curiosity, are you able to download other datasets correctly? In the past I have had issues where downloading single-cell data didn't work, but bulk RNA or DNA data worked fine.
> 
> I will upload to GEO, but I have never done it before so I need to figure out. I just filled out metadata and am figuring out how to get the data uploaded in an organized way. In the future, I will try to do both, GEO and SRA.

Hi, I have downloaded other scRNA data in SRA, it worked. For bulk RNA, I also had no problem. Today, one of my friends told me that I could try to download the data by AWS, so I'm waiting for the data transfer from NCBI to AWS with the help of NCBI, I will let you know whether if it works!
Many thanks for your time!~

**Comment by atomadam2** [comment]
Yea, for me it has been so unbelievably inconsistent. I'm worried it might be an issue with SRA because on one system I did get an error which didn't allow to even download anything. I think I'll send a call to SRA to see what might be going on on their end.

Thanks for alerting me of all this. And I apologize this is taking so long.

**Comment by kkjtmac** [comment]
> Yea, for me it has been so unbelievably inconsistent. I'm worried it might be an issue with SRA because on one system I did get an error which didn't allow to even download anything. I think I'll send a call to SRA to see what might be going on on their end.
> 
> Thanks for alerting me of all this. And I apologize this is taking so long.

Hi, finally!!! The data transferred to AWS was right and could be run by cellranger count. Each sample was the same as you listed in original format of SRA, so It should be the problem within SRA.

**Comment by xwt778** [comment]
Hi,
I tried to download the PRJNA596980 raw data from SRA, the data transferred to AWS was right and could be run by cellranger count. However, there is only one fastq file in the data SRR10744533 you uploaded.
So could you please check the files and upload the data of SRR10744533 again if convenient?
